src/main/java/unicash/logic/commands/EditTransactionCommand.java

```diff
@@ -37,16 +36,16 @@ public class EditTransactionCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the transaction identified "
```

Comment:
This should really be a StringBuilder or a custom class to handle our success messages, this pattern is way too prone to human error

Reply:
Something like having a `UsageBuilder` that takes in `Option... options` and `String usage` parameters where `Option` has `prefix`, `name`, and `isOptional` fields, this way, we can easily compose the resulting string without needing to resort to manually writing out these lengthy String concatenations for all commands, including the new ones

Reply:
As for this and the above comment, may I suggest that @jamz903 work on them alongside the `COMMAND_WORD` modifcations in v1.3b? I think it would be quite appropriate to have both `COMMAND_WORD` and `MESSAGE_USAGE` messages unified and accessible from the same place. That way, we would have a centralized location for defining/modifying user inputs and `ResultDisplay` outputs, perhaps we could also unify that with the existing `CliSyntax` as well, since everywhere that `MESSAGE_USAGE` exists, `CliSyntax.PREFIX` seems to exist as well, what do you think?